hypervm/httpdocs/lib/vps/driver/vps__xenlib.php

```diff
@@ -1192,7 +1192,7 @@ public function createConfig()
  			 *  2a. if RHEL-5/CentOS-5 or old kloxo (hostinabox) template -> use lxkernel-domU-xen kernel
 			 *  2b. if other modern linux template -> copy existing kernel modules
 			 */
-			if (is_centosfive()) {
+		    if (is_centosfive() ||is_centosseven()) {
```

Comment:
Same here. We should start Linux domUs with a kernel inside the DomU instead of a kernel that lies in the filesystem of the dom0.
Don't bother, for now, I will open a ticket and will try to clean up CentOS5 & CentOS6 related code in a near future.